applications/fishing-map/src/features/app/App.tsx

```diff
@@ -108,8 +111,11 @@ function App(): React.ReactElement {
   useEffect(() => {
     const fetchWorkspace = async () => {
       const action = await dispatch(fetchWorkspaceThunk(urlWorkspaceId as string))
-      if (fetchWorkspaceThunk.fulfilled.match(action) && action.payload?.viewport) {
-        setMapCoordinates(action.payload.viewport)
+      if (fetchWorkspaceThunk.fulfilled.match(action)) {
```

Comment:
Fixes first viewport position, we need to wait until workspace is loaded to use url or workspace params